ejb3/src/main/java/org/jboss/as/ejb3/subsystem/ApplicationSecurityDomainDefinition.java

```diff
@@ -179,8 +206,8 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
         }
     }
 
-    Predicate<String> getKnownSecurityDomainPredicate() {
-        return knownApplicationSecurityDomains::contains;
+    Function<String, ApplicationSecurityDomainConfig> getKnownSecurityDomainPredicate() {
```

Comment:
Same minor nit here, should probably rename this to "getKnownSecurityDomainFunction" now.